charts/scalardb-mcp-server/values.yaml

```diff
@@ -0,0 +1,256 @@
+# Default values for scalardb-mcp-server.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# -- Override the name of the chart.
+nameOverride: ""
+
+# -- Override the fully qualified app name.
+fullnameOverride: ""
+
+scalardbMcpServer:
+  # -- Number of replicas to deploy. Must remain 1 for now.
+  replicaCount: 1
+
+  image:
+    # -- Container image repository.
+    repository: ghcr.io/scalar-labs/scalardb-mcp-server
+    # -- Container image pull policy.
+    pullPolicy: IfNotPresent
+    # -- Overrides the image tag whose default is the chart appVersion.
+    tag: ""
+
+  # -- Secrets for pulling container images.
+  imagePullSecrets: []
+
+  # -- Annotations to add to the pod.
+  podAnnotations: {}
+
+  # -- Labels to add to the pod.
+  podLabels: {}
+
+  # -- Pod security context.
+  podSecurityContext:
+    seccompProfile:
+      type: RuntimeDefault
+
+  # -- Container security context.
+  securityContext:
+    capabilities:
+      drop:
+        - ALL
+    runAsNonRoot: true
+    allowPrivilegeEscalation: false
+
+  # -- Node selector for pod assignment.
+  nodeSelector: {}
+
+  # -- Tolerations for pod assignment.
+  tolerations: []
+
+  # -- Affinity for pod assignment.
+  affinity: {}
+
+  serviceAccount:
+    # -- Specify an existing service account name. If not specified, a new service account will be created.
+    serviceAccountName: ""
+    # -- Whether to automount the service account token.
+    automountServiceAccountToken: false
+
+  # -- Resource limits and requests for the container.
+  resources: {}
+    # limits:
+    #   cpu: 100m
+    #   memory: 128Mi
+    # requests:
+    #   cpu: 100m
+    #   memory: 128Mi
+
+  # -- The application.properties for ScalarDB MCP Server. If you want to customize application.properties, you can override this value with your application.properties.
+  # @default -- The default application.properties for ScalarDB MCP Server with OAuth 2.1 configuration.
+  applicationProperties: |
+    # ========================================
+    # MCP Server Capabilities
+    # ========================================
+    spring.ai.mcp.server.capabilities.tool=true
+    spring.ai.mcp.server.capabilities.resource=true
+    spring.ai.mcp.server.capabilities.prompt=true
+    spring.ai.mcp.server.capabilities.completion=false
+
+    # ========================================
+    # MCP Transport Configuration
+    # ========================================
+    spring.ai.mcp.server.stdio=false
+    spring.ai.mcp.server.protocol=STREAMABLE
+    spring.main.web-application-type=servlet
+    spring.main.banner-mode=console
+    server.port=${SERVER_PORT:8080}
```

Comment:
It's not helm chart stuff, but I have one question.

Is the `8080` port the default port of ScalarDB MCP Server?

If yes, I think it conflicts with many other tools or services.

If I remember correctly, when we released Scalar Manager, we discussed the default port to avoid conflict as long as possible and decided to use `13000` as a default port.

Similar to that, I think it might be better to discuss the default port of ScalarDB MCP Server.

@feeblefakie 
What do you think?